File: Herencia/ejercicio 1/cliente.py
from persona import Persona

class Cliente(Persona):
    def __init__(self, nombre: str, apellido: str, ci: str, nroCompra: str, idCliente: str):
        super()._init_(nombre, apellido, ci)
        self.nroCompra = nroCompra
        self.idCliente = idCliente

    def mostrar_datos(self) -> str:
        return f"Cliente: {self.nombre} {self.apellido}, CI: {self.ci}, NroCompra: {self.nroCompra}, ID: {self.idCliente}"

File: Herencia/ejercicio 1/Cliente.java
package herencia;
public class Cliente extends Persona {
    private String nroCompra;
    private String idCliente;

    public Cliente(String nombre, String apellido, String ci, String nroCompra, String idCliente) {
        super(nombre, apellido, ci);
        this.nroCompra = nroCompra;
        this.idCliente = idCliente;
    }

    @Override
    public String mostrarDatos() {
        return "Cliente: " + nombre + " " + apellido + ", CI: " + ci +
               ", NroCompra: " + nroCompra + ", ID: " + idCliente;
    }
}